_episodes/07-github.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Remotes in GitHub
+title: "Remotes in GitHub"
+keypoints:
```

Comment:
I think keypoints, questions, and objectives each need to be in quotes in order to render properly.

Reply:
This is now fixed.

Reply:
This is now fixed.